packages/@aws-cdk/aws-apigateway/README.md

```diff
@@ -193,6 +193,7 @@ new apigateway.StepFunctionsRestApi(this, 'StepFunctionsRestApi', {
   headers: true,
   path: false,
   querystring: false,
+  authorizer: false,
```

Comment:
This part was actually not implemented. Please take a look at the build failure.

Reply:
My bad -- I added it to the Integration but not the RestApi